### PR TITLE
clustergram 0.8.1

### PR DESCRIPTION
**Pull request by psteyer**
clustergram 0.8.1 :snowflake:

**Destination channel:** Snowflake 

### Links

- [PKG-4542](https://anaconda.atlassian.net/browse/PKG-4542) 
- [Upstream repository](https://github.com/martinfleis/clustergram/tree/v0.8.1)

### Explanation of changes:

- initial recipe creation and package build
- added `openblas` force due to `Fatal Python error: Segmentation fault` on `osx-64`

[PKG-4542]: https://anaconda.atlassian.net/browse/PKG-4542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    # mkl variant pulls in intel-openmp which conflicts with llvm-openmp. i.e. force to use openblas variant of numpy, scipy, etc.
+    - blas=*=openblas  # [osx and x86_64]
+test:
+  source_files:
+    - clustergram/test_clustergram.py
+  imports:
+    - clustergram
+  commands:
+    - pip check
+    - pytest -v clustergram/test_clustergram.py
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - pytest -v --pyargs clustergram
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - blas=*=openblas  # [osx and x86_64]
+  run:
+    - python
+    - pandas
+    - numpy
+    - matplotlib-base
+    # mkl variant pulls in intel-openmp which conflicts with llvm-openmp. i.e. force to use openblas variant of numpy, scipy, etc.
+    - blas=*=openblas  # [osx and x86_64]
+test:
+  source_files:
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - clustergram/test_clustergram.py
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

**Review by cbouss — COMMENTED** [review]
Since the tests are included in the package, you don't need to include them a second time with source_files

**Review by boldorider4 — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  description: |
+    This is a Python implementation, originally based on Tal's script, written 
+    for scikit-learn and RAPIDS cuML implementations of K-Means, Mini Batch K-Means 
+    and Gaussian Mixture Model (scikit-learn only) clustering, plus hierarchical/agglomerative 
+    clustering using SciPy.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/martinfleis/clustergram
+  doc_url: https://clustergram.readthedocs.io/en/stable/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://clustergram.readthedocs.io/
```
To address the `conda-linter` warning message

**Review by skupr-anaconda — APPROVED** [review]

**Review by cbouss — APPROVED** [review]

**Pull request merged**